pallets/gear/src/schedule.rs

```diff
@@ -733,12 +733,27 @@ fn cost_instr_no_params_with_batch_size(w: fn(u32) -> Weight) -> u32 {
 
 #[inline]
 fn cost_instr<T: Config>(w: fn(u32) -> Weight, num_params: u32) -> u32 {
-    type W<T> = <T as Config>::WeightInfo;
+    cost_instr_no_params_with_batch_size(w)
+        .saturating_sub(cost_i64const::<T>().saturating_mul(num_params))
+}
 
-    cost_instr_no_params_with_batch_size(w).saturating_sub(
-        (cost_instr_no_params_with_batch_size(W::<T>::instr_i64const) / 2)
```

Comment:
Cannot remove / 2 from calculation of `i64const`  weight (we estimate `i64cost = `instr_call_const` - `instr_call` and it really inaccurate), because it actually make it closer to what it should be (x86-64 `mov`).